### PR TITLE
fix(tile): correct icon slot visibility

### LINKED ISSUE #1503: [bug] `<rh-tile>`: incorrectly hides icon slot when icon attribute is undefined
`<rh-tile>` will hide the `icon` slot when the `icon` attribute is undefined.  However if we want to slot a icon using `` this slotted content will be hidden.  The only way to use the icon slot in the current state is to use a `pf-icon` from the set `far` as there is also no way to change the set, you can set the icon to a string value that isn't a pf-icon but that is not ideal as you are loading an empty `pf-icon` component in the shadowroot as a tradeoff. 

Codepen that demos the issue:

https://codepen.io/zeroedin/pen/mdgRzxq

/cc @ocean-rh

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Fixed template logic to correctly display slotted content when the icon attribute is undefined.

Closes #1503 

## Testing Instructions

1. View deploy preview icon demo

### CODE REVIEW

File: elements/rh-tile/rh-tile.ts
[the remark this answers is not in the document]
Comment (zeroedin) [reply elements/rh-tile/rh-tile.ts:1]: ooh good call

File: elements/rh-tile/rh-tile.ts [hunk 1]
```diff
@@ -160,6 +164,11 @@ export class RhTile extends LitElement {
     this.addEventListener('click', this.#onClick);
   }
 
+  connectedCallback(): void {
+    super.connectedCallback();
```
Comment (bennypowers) [inline elements/rh-tile/rh-tile.ts:168]: Just call this in render. No need for class private state

**Review by bennypowers — CHANGES_REQUESTED** [review]
Please inline the controller call to render

**Review by bennypowers — APPROVED** [review]
little graphics through markup

**Pull request merged**